org.falcao.sup.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Name=Sup
+Icon=whatsapp
+Exec=/app/sup
```

Comment:
it's never a full path. It's should be the name of the command and the command be in the PATH.